### PR TITLE
NAS-109029 / 21.04 / Allow user to configure GPU in Launch Docker image wizard in UI / NAS-109029

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/pages/common/entity/utils.ts [hunk 1]
```diff
@@ -265,4 +265,183 @@ export class EntityUtils {
...
+          operator: relation[1],
+          value: relation[2],
+        }]
+      };
+    });
+
+    return result;    
+  }
+
+  parseSchemaFieldConfig(schemaConfig, parentName=null, parentIsList=false) {
```
Comment (ryousaf) [inline src/app/pages/common/entity/utils.ts:289]: We should have classes and types where feasible for cleaner implementation and maintenance

Comment (AlexKarpov98) [reply src/app/pages/common/entity/utils.ts]: Regarding using class, they are all dynamic so we can't know its structure beforehand.
Also it is related with dynamic chart release and all data is from schema that middleware sends and it has unexpected structure.
That's why we can only use `any `or `object` here.

**Review by ryousaf — APPROVED** [review]
LGTM

**Review by ryousaf — APPROVED** [review]
LGTM

**Pull request merged**